tests/integration/data_sources_and_expectations/test_misconfigured_expectations.py

```diff
@@ -67,7 +67,7 @@ def test_pandas(self, batch_for_datasource) -> None:
     def test_bigquery(self, batch_for_datasource) -> None:
         self._assert_misconfiguration(
             batch_for_datasource=batch_for_datasource,
-            exception_message="No matching signature for operator * for argument types: FLOAT64, STRING",  # noqa: E501
```

Comment:
Could you help me understand this change better? Would we still be missing the operator for `*` for `STRING`?